### PR TITLE
blog: add post on sig-txn reading group

### PR DESCRIPTION
**Pull request by nrc**
### What is changed?

Adds a blog post on the sig-txn reading group

### Any related PRs or issues?

no

### Which version does your change affect?

n/a

PTAL @dcalvin

### CODE REVIEW

File: content/blog/sig-txn-reading-group-nov-20.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: SIG-txn reading group, Nov + Dev 2020
+author: Nick Cameron
+date: 2020-11-04
+tags: ['Research', 'Transactions', 'Community']
```
Comment (dcalvin) [inline content/blog/sig-txn-reading-group-nov-20.md:5]:
```suggestion
tags: ['Research', 'Transactions', 'Community', `SIG`]
```

**Review by dcalvin — APPROVED** [review]
LGTM.

**Comment by dcalvin** [comment]
@nrc missing the DCO

**Comment by nrc** [comment]
@dcalvin fixed, could you merge this please?

**Pull request merged**